demo/campaign/src/components/IndentedSectionGroup.tsx

```diff
@@ -7,18 +7,31 @@ export interface IndentedSectionGroupProps extends React.ComponentProps<typeof M
     cssClass?: string;
 }
 
-const CONTENT_WIDTH = theme.mailSize.contentWidth - theme.mailSize.contentSpacing * 2;
+export const indentedSectionGroupStyles = (
+    <MjmlStyle>{`
+      @media (max-width: 759px) {
+        .indented-section__group {
+          width: 100% !important;
+          max-width: 592px;
+        }
+      }
+    `}</MjmlStyle>
+);
+
+const CONTENT_WIDTH = theme.mailSize.contentWidth;
 
 export const IndentedSectionGroup = ({ children, cssClass, ...restProps }: IndentedSectionGroupProps) => {
     return (
         <MjmlSection
-            cssClass={`${cssClass || ""}`.trim()}
+            cssClass={(`${cssClass || ""}`.trim(), "indented-section")}
             paddingLeft={theme.mailSize.contentSpacing}
             paddingRight={theme.mailSize.contentSpacing}
             backgroundColor={theme.colors.background.content}
             {...restProps}
         >
-            <MjmlGroup width={CONTENT_WIDTH}>{children}</MjmlGroup>
+            <MjmlGroup width={CONTENT_WIDTH} cssClass="indented-section__group">
```

Comment:
I'd just use the theme variable directly here, without defining an additional variable. 

```suggestion
            <MjmlGroup width={theme.mailSize.contentWidth} cssClass="indented-section__group">
```